### PR TITLE
Add lava tile as svg and to title.js Fixes #418

### LINKED ISSUE #418: Add lava tile
### Expected results
Lava tile in the game

### Actual results
No lava tile in the game

### Steps to reproduce the issue
Create lava.svg and add it to code

## Would you like to work on this
Yes 😄

### PR DESCRIPTION
**Pull request by kubami9**
Add lava tile as svg and to title.js Fixes #418

### When you change [things like this in brackets] please remember to delete brackets after you change it.

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [x] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request
Add lava tile as svg and to `title,js`

- [Preview Link](https://kubami9.github.io/labyrinth-km-contribute/#)
- Fixes`#418`

# The problem(s) I want to solve or The facilitie(s) I want to improve is/are,
Add lava tile as svg and to `title,js`

### My steps to solve it,
Add lava tile as svg and to `title,js`

### CODE REVIEW

File: js/tiles.js [hunk 1]
```diff
@@ -1104,4 +1104,33 @@ const desert = {
...
+            return true;
+        },
+        canLeaveToTheBottom() {
+            return true;
+        },
+        canLeaveToTheRight() {
+            return false;
+        },
+        visit: function() {
+            swal({
```
Comment (niccokunzmann) [inline js/tiles.js]: We have an API for that because the dialog functionality might change in the future.

Comment (niccokunzmann) [reply js/tiles.js]: see readme.md

**Comment by niccokunzmann** [comment]
Thanks for adding this. Is the lava tile your own creation? If not, please show that you have the rights to license it under AGPL.

**Comment by kubami9** [comment]
@niccokunzmann it's my own creation, to create the tile I included edited image in a shape

**Comment by niccokunzmann** [comment]
Yes, but is the lava image your work? That what we can see?

**Comment by niccokunzmann** [comment]
Did you copy it from somewhere?

**Comment by niccokunzmann** [comment]
It looks so authentic so I would like to ask.

**Comment by kubami9** [comment]
Ohh, the lava image, it's from [here,](https://pixabay.com/en/lava-cracked-background-fire-656827/) only experimented with some colors and filters so it looks more similar to other lava images I found which unfortunately weren't free (I wanted lava to be bright, orange). If needed, I can quickly recreate this tile with original non-edited image, it under _CC0 Creative Commons_

**Comment by niccokunzmann** [comment]
Cool. This is what I wanted to know. Great. Thanks. So, since right now, others are also adding pictures from pixbay, https://github.com/fossasia/labyrinth/pull/420#issuecomment-357014065 
If you move the picture to /tiles/rooms/floor/pixbay/, you can use the license of theirs, too.

**Comment by niccokunzmann** [comment]
CC0 means: public domain where possible elsewhere CC-BY e.g. in Germany.

**Comment by niccokunzmann** [comment]
Would that be ok? I will merge either way.

**Comment by kubami9** [comment]
Sure! 😄

**Comment by kubami9** [comment]
@niccokunzmann So it's ready to merge, right? I don't have to edit anything?

**Comment by niccokunzmann** [comment]
You do not have to but we need an edit and you can do it. Moving your tile to a pixbay folder and adding the license.

**Comment by kubami9** [comment]
Okay, I'd like to work on it :)

**Pull request merged**